lists/arraylist/arraylist.go

```diff
@@ -43,10 +42,10 @@ func New[T comparable](values ...T) *List[T] {
 
 // Add appends a value at the end of the list
 func (list *List[T]) Add(values ...T) {
+	l := len(list.elements)
 	list.growBy(len(values))
-	for _, value := range values {
-		list.elements[list.size] = value
-		list.size++
+	for i := range values {
+		list.elements[l+i] = values[i]
```

Comment:
This can just be replaced with `append`

Reply:
Based on the benchmark results, it seems append is somehow slower, so dropping this comment